resources/views/profile/index.blade.php

```diff
@@ -0,0 +1,61 @@
+
+<x-app-layout>
+    <div class="min-h-screen bg-gradient-to-tr from-sky-500 to-orange-400">
+    <div class="flex justify-center ">
+        <div class="bg-white/10 backdrop-blur-md border border-white/50 rounded-xl shadow-lg overflow-hidden w-full mt-28"  id="container">
+
+            <!-- Header Section -->
+            <div class="relative flex justify-center pt-8">
+
+                <img class=" border-2 border-[#76A8B2] -bottom-10 left-8 w-40 h-40 rounded-full "
+                     src="http://picsum.photos/seed/{{rand(62, 999999)}}/160/160/" alt="">
```

Comment:
Here we need profile picture from user, or if user don't added image, use default avatar